Sources/SyntaxGen/Generator.swift

```diff
@@ -12,6 +12,13 @@ extension FileHandle: TextOutputStream {
     }
 }
 
+extension String {
+  func camelCased() -> String {
```

Comment:
We have `.lowercaseFirstWord` for this

Reply:
Not defined anywhere I can find.

Reply:
Ah, sorry, it’s lowercaseFirstLetter in String+Helpers